### DISCUSSION TITLE
[Feral Druid] Why would Hekili recommend Rip instead of Primal Wrath when there are 3+ targets?

### DISCUSSION THREAD

**Discussion by groooooooooooot** [Hekili/hekili#968] [General]
Im trying to understand in what scenario Rip would ever be recommended over Primal Wrath with 3 or more targets present and in range.

I'm using the 4 target dummies in Orgrimmar for example, and Hekili is consistently recommending Rip even though all 4 dummies are in range and already bleeding (thrash).  I cant imagine there is any scenario where Rip on 1 target is a better option than Primal Wrath on 4 targets?

Hekili is a clean install of the latest version.  Display mode is set to Auto, but even when set to AoE Primal Wrath is not being suggested.  I am basically just primal wrathing whenever i see the Rip icon in this scenario but its still odd to me that Rip would be recommended.

**Comment by groooooooooooot** [comment]
Further information... this seems to be related to having Circle of Life Death legendary equipped.  When I remove the Circle legendary and replace with regular gear, Primal Wrath is now recommended as expected.  I'm thinking this is a bug in the feral applet specific to this legendary.

**Reply by Hekili** [reply]
Issue being worked out per your ticket.
